{
  "uid" : "b571ac5635cac261",
  "name" : "Can find connect button on Jetpack page",
  "fullName" : "Jetpack pre-connection.Can find connect button on Jetpack page",
  "historyId" : "e328479f6c0e60fd904d4b6c94c7542b",
  "time" : {
    "start" : 1599564035675,
    "stop" : 1599564063380,
    "duration" : 27705
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "failed",
  "statusDetails" : {
    "message" : "TypeError [ERR_INVALID_URL]: Invalid URL: null",
    "trace" : "TypeError [ERR_INVALID_URL]: Invalid URL: null\n                    at onParseError (internal/url.js:241:17)\n                    at new URL (internal/url.js:319:5)\n                    at _callee5$ (/home/travis/build/Automattic/jetpack/tests/e2e/lib/flows/jetpack-connect.js:125:15)\n                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)\n                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)\n                    at Generator.prototype.(anonymous function) [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)\n                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)\n                    at /home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7\n                    at new Promise (<anonymous>)",
    "flaky" : false
  },
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusDetails" : {
      "message" : "TypeError [ERR_INVALID_URL]: Invalid URL: null",
      "trace" : "TypeError [ERR_INVALID_URL]: Invalid URL: null\n                    at onParseError (internal/url.js:241:17)\n                    at new URL (internal/url.js:319:5)\n                    at _callee5$ (/home/travis/build/Automattic/jetpack/tests/e2e/lib/flows/jetpack-connect.js:125:15)\n                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)\n                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)\n                    at Generator.prototype.(anonymous function) [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)\n                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)\n                    at /home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7\n                    at new Promise (<anonymous>)",
      "flaky" : false
    },
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "c7ceb1470286f73b",
      "name" : "Test failed: Jetpack pre-connection :: Can find connect button on Jetpack page",
      "source" : "c7ceb1470286f73b.png",
      "type" : "image/png",
      "size" : 32655
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Jetpack pre-connection"
  }, {
    "name" : "testClass",
    "value" : "Jetpack pre-connection"
  }, {
    "name" : "testMethod",
    "value" : "Can find connect button on Jetpack page"
  }, {
    "name" : "package",
    "value" : "Jetpack pre-connection"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "extra" : {
    "tags" : [ ]
  },
  "source" : "b571ac5635cac261.json",
  "statusDetailsSafe" : {
    "message" : "TypeError [ERR_INVALID_URL]: Invalid URL: null",
    "trace" : "TypeError [ERR_INVALID_URL]: Invalid URL: null\n                    at onParseError (internal/url.js:241:17)\n                    at new URL (internal/url.js:319:5)\n                    at _callee5$ (/home/travis/build/Automattic/jetpack/tests/e2e/lib/flows/jetpack-connect.js:125:15)\n                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)\n                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)\n                    at Generator.prototype.(anonymous function) [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)\n                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)\n                    at /home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7\n                    at new Promise (<anonymous>)",
    "flaky" : false
  },
  "statusMessage" : {
    "empty" : false,
    "present" : true
  },
  "parameterValues" : [ ]
}